docs/mast/mast.rst

```diff
@@ -868,6 +868,97 @@ To access sector information at a particular location there is  `~astroquery.mas
+                >>> hdulist[0].info()
+                Filename: <class '_io.BytesIO'>
+                No.    Name      Ver    Type      Cards   Dimensions   Format
+                  0  PRIMARY       1 PrimaryHDU      11   ()
+                  1  CUTOUT        1 ImageHDU       177   (5, 5)   float32
+                  2  CUTOUT        1 ImageHDU       177   (5, 5)   float32
+                  3  CUTOUT        1 ImageHDU       177   (5, 5)   float32
+
+
+The ~astroquery.mast.ZcutClass.download_cutouts function takes a coordinate and cutout size (in pixels or
+an angular quantity) and downloads the cutout target pixel file(s) as either fits files or image (png/jpg)
+files.
+
+If a given coordinate appears in more than one Zcut survey, a target pixel file will be produced for each
+survey. If the cutout area overlaps more than one camera or ccd a target pixel file will be produced for each
+one.
+
+.. code-block:: python
+
+                >>> from astroquery.mast import Zcut
+                >>> from astropy.coordinates import SkyCoord
+
+                >>> cutout_coord = SkyCoord(189.49206, 62.20615, unit="deg")
+                >>> manifest = Zcut.download_cutouts(coordinates=cutout_coord, size=[200, 300], units="px")
+                Downloading URL https://mast.stsci.edu/zcut/api/v0.1/astrocut?ra=189.49206&dec=62.20615&y=200&x=300&units=px&format=fits to ./zcut_20201202132247.zip ... [Done]
+
+                >>> print(manifest)
+                                                Local Path
+                -------------------------------------------------------------------------
+                ./candels_gn_30mas_189.492060_62.206150_300.0pix-x-200.0pix_astrocut.fits
+
+
+.. code-block:: python
+
+                >>> from astroquery.mast import Zcut
+                >>> from astropy.coordinates import SkyCoord
+
+                >>> cutout_coord = SkyCoord(189.49206, 62.20615, unit="deg")
+                >>> manifest = Zcut.download_cutouts(coordinates=cutout_coord, size=[200, 300], units="px", form="jpg")
+                Downloading URL https://mast.stsci.edu/zcut/api/v0.1/astrocut?ra=189.49206&dec=62.20615&y=200&x=300&units=px&format=jpg to ./zcut_20201202132453.zip ... [Done]
+
+                >>> print(manifest)
+                                                                Local Path
+                ---------------------------------------------------------------------------------------------------------
+                 ./hlsp_candels_hst_acs_gn-tot-30mas_f606w_v1.0_drz_189.492060_62.206150_300.0pix-x-200.0pix_astrocut.jpg
+                 ./hlsp_candels_hst_acs_gn-tot-30mas_f814w_v1.0_drz_189.492060_62.206150_300.0pix-x-200.0pix_astrocut.jpg
+                ./hlsp_candels_hst_acs_gn-tot-30mas_f850lp_v1.0_drz_189.492060_62.206150_300.0pix-x-200.0pix_astrocut.jpg
+
+
+Survey information
+------------------
+
+To access survey information at a particular location there is ~astroquery.mast.ZcutClass.get_surveys.
```

Comment:
To access survey information at a -> To list the available deep field surveys at a